CHANGES.rst

```diff
@@ -6,6 +6,9 @@ There's a frood who really knows where his towel is.
 1.0a13 (unreleased)
 ^^^^^^^^^^^^^^^^^^^
 
+- Implement CSRF protection (closes `590`_).
```

Comment:
@rodfersou this is not enough; there are implications with this change: what would happen if the user don't have `plone4.csrffixes` installed?